README.md

```diff
@@ -1070,6 +1070,7 @@ Inspired by [awesome-php](https://github.com/ziadoz/awesome-php).
 
 *Frameworks for developing serverless Python code.*
 
+* [AWS Lambda Powertools](https://github.com/gyft/aws-lambda-powertools-python) - A suite of utilities for AWS Lambda functions to ease adopting best practices such as tracing, structured logging, custom metrics async, parameters and secrets management, idempotency, and more.
```

Comment:
Why not the original one? This fork has nothing additional on the original repo.

Reply:
Oops yes, i used my fork of it :). Fixed now.